src/main/kotlin/no/nav/syfo/db/PlanlagteVarslerDAO.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.syfo.db
+
+import no.nav.syfo.domain.PlanlagtVarsel
+import java.sql.Date
+import java.sql.Timestamp
+import java.time.LocalDateTime
+import java.util.*
+
+fun DatabaseInterface.storePlanlagtVarsel(planlagtVarsel: PlanlagtVarsel, planlagtVarselJSON: String): UUID {
+
+    // TODO: Bør orgnummer ligge utenfor 'data'-feltet i JSON (e.g. obligatorisk felt i alle varsel-meldinger i varsel-planlegger-topic) ?
```

Comment:
Jeg tror du er inne på noe her. Jeg har også tenkt på om vi kanskje må gjøre det samme for varselbus'en.